GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/BasicFileSystemTests.cs

```diff
@@ -918,6 +919,23 @@ public void CreateDirectoryInheritsParentDirectoryAttributes(string parentFolder
             FileSystemRunner.DefaultRunner.DeleteDirectory(parentDirectoryPath);
         }
 
+        [TestCase]
+        [Category(Categories.MacOnly)]
+        public void RunPythonExecutable()
+        {
+            string pythonDirectory = Path.Combine(this.Enlistment.RepoRoot, "Test_Executable");
+            string pythonExecutable = Path.Combine(pythonDirectory, "python_wrapper.sh");

```

Comment:
Is there a reason the python script launch is wrapped in a shell script? (I’m just curious)

Reply:
Exellent question.  While calling test.py on the command line repros the issue, calling it from C# does not.  C# does something to hydrate it before execution.  If I had to guess it's reading attributes.  By calling via a wrapper I can repro the issue.  

Side note, the wrapped added an extra level of fun to reproing in the unit test.

Reply:
OK, that's a good reason! Perhaps we'd like to document that in a comment in the test?

Reply:
@pmj excellent point I'll add that documentation!